Jenkinsfile

```
@@ -278,7 +295,7 @@ pipeline {
 											done
 										fi
 									fi
-									cp *.$binariesExtension "${WORKSPACE}/eclipse.platform.swt/binaries/org.eclipse.swt.${PLATFORM}/"
+									cp *.$binariesExtension "${WORKSPACE}/eclipse.platform.swt/binaries/org.eclipse.swt.${PLATFORM/gtk4/gtk}/"
```

As somebody not using Linux sh/bash in my everyday life I had a hard time to understand this 'replacing' variable expansion. In order to make this pipeline a bit more understandable people who are not experts in this area I suggest to do the replacement in the corresponding if-block above. This also groups the changes for this issue a bit closer.
See also my other comment above.

```suggestion
									cp *.$binariesExtension "${WORKSPACE}/eclipse.platform.swt/binaries/org.eclipse.swt.${PLATFORM}/"
```